api/client.yaml

```diff
@@ -1513,6 +1513,249 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/moov-io/base/master/api/common.yaml#/components/schemas/Error'
+  /customers/{customerID}/representatives:
+    post:
+      tags: [Customers]
```

Comment:
After https://github.com/moov-io/customers/pull/272 (which is done to cleanup api.moov.io) I'm thinking we want this on a "Customer Representatives" tag. Thoughts?

Reply:
What would be the benefit of using a `Customer Representatives` tag?  In my mind, this is still related to customers and things that pertain to customers, but I am open to using `Customer Representatives` as the tag

Reply:
Changed to `Representatives`